sherpa/plot/pylab_backend.py

```diff
@@ -240,8 +244,10 @@ def plot(x, y, yerr=None, xerr=None, title=None, xlabel=None, ylabel=None,
          barsabove=_errorbar_defaults['barsabove'],
          xlog=False,
          ylog=False,
+         label=None,
          linestyle='solid',
          linecolor=None,
+         linewidth=None,
          drawstyle='default',
```

Comment:
Why not:
```
def plot(x, y, yerr=None, xerr=None, title=None, xlabel=None, ylabel=None,
         overplot=False, clearwindow=True,
         xerrorbars=False,
         yerrorbars=False,
         ecolor=_errorbar_defaults['ecolor'],
         capsize=_errorbar_defaults['capsize'],
         barsabove=_errorbar_defaults['barsabove'],
         xlog=False,
         ylog=False, **kwargs)
```

Reply:
It's not that easy. We set up the defaults knowing what the keywords should be. This could be relaxed, but it needs a careful think about.

Having another backend would help identify things we want to keep and those we can change :-)